papers/Mathematics/paper.Rmd

```diff
@@ -13,6 +13,9 @@ author:
   - name: Dominique Makowski
     affil: 4
     orcid: 0000-0001-5375-9967
+  - name: Brenton M. Wiernik
```

Comment:
Also if nobody opposes (@bwiernik @rempsyc) I wouldn't mind being last here... the academic path won't grind itself 🥲

Reply:
No preferences from me

Reply:
Sure, feel free to change it

Reply:
Go for it!